src/diffusers/models/attention_utils.py

```diff
@@ -0,0 +1,7 @@
+from torch import BoolTensor, FloatTensor
+import torch
+
+def mask_to_bias(mask: BoolTensor, dtype: torch.dtype) -> FloatTensor:
```

Comment:
We don't like to create one-liner functions as this reduces readability - could we maybe not abstract this away with a `mask_to_bias` function, but instead leave the line `(1 - mask.to(dtype=dtype) * -10000.0)` when needed (which is just two times)

Reply:
it may look like a single-use one-line function in your codebase, but in my codebase I want to convert my masks to biases the same way diffusers does. how can I code-share?

Reply:
Makes sense to me that other libraries would want to do the same.